tests/integration/frameworks/test_pytorch_impl.py

```diff
@@ -120,3 +122,20 @@ def test_pytorch_runner_with_partial_kwargs(modelstore, bias_pair):
     # tensor to float may introduce larger errors, so we bump rel_tol
     # from 1e-9 to 1e-6 just in case
     assert math.isclose(res1 - res2, bias1 - bias2, rel_tol=1e-6)
+
+
+@pytest.mark.parametrize("batch_axis", [0, 1])
+def test_pytorch_container(batch_axis):
```

Comment:
could you add a test that loads a custom runner with PyTorch tensor input, and test both its `run` and `run_batch` method?

Reply:
Can we test with current pytorch runner? I push a draft test in the new commit

Reply:
that is great, it would be great to also add a minimal test case to test data container separately, e.g. create a dummy runner class with PyTorch tensor input type and test its behavior.  I will go ahead and merge this PR, feel free to add that in follow up PR.

```python
class DummpyPyTorchRunner(Runner):
   def _run_batch(self, input_tensor):
          ...
```